example/particles/particles.cpp

```diff
@@ -86,11 +90,11 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   std::string swarm_name = "my particles";
   Metadata swarm_metadata({Metadata::Provides, Metadata::None});
   pkg->AddSwarm(swarm_name, swarm_metadata);
-  Metadata real_swarmvalue_metadata({Metadata::Real});
+  Metadata real_swarmvalue_metadata({Metadata::Real, Metadata::Particle});
```

Comment:
I am not sure I understand why `Metadata::Particle` needs to be included here. Shouldn't any swarm be `Metadata::Particle` by default?

Reply:
nm, just saw Jonah's comment below and the response.